opossum-tool/OpossumUI
# feat: move autocomplete suggestions calculation to web worker

### Summary of changes

- adjust CSP to allow web worker blobs
- create signals web worker with hydration mechanism for expensive calculations

NOTE: There will be two more PRs to refactor the other existing web workers with this new approach in order to reduce the memory footprint on the application.

### Context and reason for change

#2384 

### How can the changes be tested

Check that going between resources in the audit view's resource browser is not feeling slow.
